tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -199,8 +199,10 @@ def test_positive_smoke(self):
                     common_locators['alert.success']
                 ))
             # Add puppet-module to content-view
-            self.content_views.add_puppet_module(
-                cv_name, 'httpd', filter_term='Latest')
+            if not bz_bug_is_open(1297308):
```

Comment:
Problem here is that we would not fail the test, only bypass this step. This would "mask" the failure for us, though the other change also found in the PR would flag it. I wonder if we should skip the entire test instead?